### PR TITLE
KOGITO-4972: Enable security on infinispan container in process-usert…

### PR DESCRIPTION
**Pull request by nmirasch**
…asks-with-security-oidc-quarkus-with-console example

jira: https://issues.redhat.com/browse/KOGITO-4972

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @radtriste I have added authentication on infinispan to share application.properties and don't break the native build.
The only way found to make the sample work was using the nightly version of data-index. Is there any issue with keeping it nightly?
Could you take a look? Thanks!

File: process-usertasks-with-security-oidc-quarkus-with-console/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -69,7 +71,7 @@ services:
 
   data-index:
     container_name: data-index
-    image: quay.io/kiegroup/kogito-data-index-infinispan:${KOGITO_VERSION}
+    image: quay.io/kiegroup/kogito-data-index-infinispan-nightly:${KOGITO_VERSION}
```
Comment (radtriste) [inline process-usertasks-with-security-oidc-quarkus-with-console/docker-compose/docker-compose.yml]: please set back to prod image ;)
https://issues.redhat.com/browse/KOGITO-4609 is supposed to enable nightlies in master and then to replace once released

**Review by radtriste — CHANGES_REQUESTED** [review]

File: process-usertasks-with-security-oidc-quarkus-with-console/docker-compose/docker-compose.yml (hunk 1 above)
Comment (cristianonicolai) [reply process-usertasks-with-security-oidc-quarkus-with-console/docker-compose/docker-compose.yml]: @nmirasch yeah we shouldnt use nightly here. Now that 1.5 has been released, using latest doesnt fix the issue?

**Review by radtriste — APPROVED** [review]

File: process-usertasks-with-security-oidc-quarkus-with-console/docker-compose/docker-compose.yml (hunk 1 above)
Comment (nmirasch) [reply process-usertasks-with-security-oidc-quarkus-with-console/docker-compose/docker-compose.yml]: @cristianonicolai The 'latest' I'm getting is 5 days old and with that, this example in my environment is not working properly.  I've restored the image reference to use the latest, but if you can give this a chance and confirm that is something that only happens in my environment, would be great.

**Review by cristianonicolai — APPROVED** [review]

**Comment by cristianonicolai** [comment]
@nmirasch it works fine if versions are aligned, snapshot and nightly or 1.5.0.Final for both, otherwise it fails because of this fix: https://github.com/kiegroup/kogito-apps/pull/737. Basically in that context we have different attributes sent in the payload. This should work fine once it goes into stable branch.

**Pull request merged**